test/integration/crcsuite/crcsuite.go

```diff
@@ -5,6 +5,7 @@ package crcsuite
 import (
 	"crypto/tls"
 	"fmt"
+	"github.com/cucumber/messages-go/v10"
```

Comment:
Maybe we should move this to what's now line 20 (the other paragraph of imports).